<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Тест плагина</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayoutPlugin"/>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btnStart">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="test_plugin.qrc">
         <normaloff>:/img/startTask.png</normaloff>:/img/startTask.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnOptions">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="test_plugin.qrc">
         <normaloff>:/img/options.png</normaloff>:/img/options.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="test_plugin.qrc"/>
 </resources>
 <connections/>
</ui>
